client/my-sites/themes/thanks-modal.jsx

```diff
@@ -175,6 +176,7 @@ export default connect(
 			detailsUrl: site && getThemeDetailsUrl( state, currentThemeId, site.ID ),
 			customizeUrl: site && getThemeCustomizeUrl( state, currentThemeId, site.ID ),
 			forumUrl: site && getThemeForumUrl( state, currentThemeId, site.ID ),
+			visitSiteUrl: site.URL + ( isJetpackSite( state, site.ID ) ? '' : '?next=customize' ),
```

Comment:
You need a `site &&` check here (like on the other lines) or this will break if no site is currently selected (see failing CircleCI tests).